tooling/karaf-maven-plugin/src/main/java/org/apache/karaf/tooling/features/InstallKarsMojo.java

```diff
@@ -562,6 +562,11 @@ private void installBundle(Bundle bundle) throws Exception {
             if (bundleLocation.startsWith("war:")) {
                 bundleLocation = bundleLocation.substring("war:".length());
             }
+            // checked all known prefixes, clean a possible custom one wrapping mvn:
+            int mvnIndex = bundleLocation.indexOf("mvn:");
+            if (mvnIndex > 0) {
+                bundleLocation = bundleLocation.substring(mvnIndex);
+            }
             File bundleFile;
             if (bundleLocation.startsWith("mvn:")) {
```

Comment:
With your change, it means that this is never reach. Can you add a rationale about this change ? I'm not sure I understand exactly the purpose.

Reply:
It will reach it, because the `mvn:` part of the location will remain. All it did was to strip away the custom wrapper.

So `my-deployer:mvn:org.webjars.bower/angular/1.5.8` will become `mvn:org.webjars.bower/angular/1.5.8` making it eligible to enter the other if conditions.